sass/chosen.scss

```diff
@@ -137,7 +137,7 @@ $chosen-sprite-retina: url('chosen-sprite@2x.png') !default;
   }
   &.chosen-container-single-nosearch .chosen-search {
     position: absolute;
```

Comment:
This property is no longer necessary. It's a hold over from when we used to do `left: -9999px; position: absolute;` to hide the search box.